ocaml/xapi/xapi_pci_helpers.ml

```diff
@@ -172,3 +172,36 @@ let get_host_pcis () =
 let igd_is_whitelisted ~__context pci =
   let vendor_id = Db.PCI.get_vendor_id ~__context ~self:pci in
   List.mem vendor_id !Xapi_globs.igd_passthru_vendor_whitelist
+
+let update_dom0_access ~__context ~pci ~action =
+  let db_new =
```

Comment:
The name suggests this has something to do with a database but it's the state of the PCI visibility. I suggest to find a better name. I now realise this is coming from existing code - would still improve it.